Composer/packages/client/src/store/reducer/index.ts

```diff
@@ -317,5 +327,7 @@ export const reducer = createReducer({
   [ActionTypes.GET_ENDPOINT_SUCCESS]: updateRemoteEndpoint,
   [ActionTypes.REMOVE_RECENT_PROJECT]: removeRecentProject,
   [ActionTypes.EDITOR_SELECTION_VISUAL]: setVisualEditorSelection,
+  [ActionTypes.ONBOARDING_ADD_COACH_MARK_REF]: onboardingAddCoachMarkRef,
+  [ActionTypes.ONBOARDING_SET_COMPLETE]: onboardingSetComplete,
   [ActionTypes.EDITOR_CLIPBOARD]: setClipboardActions,
-});
+} as { [type in ActionTypes]: ReducerFunc });
```

Comment:
We want to not cast this here because we get strong type checking that each ActionType has a corresponding reducer action.